spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  describe 'finds a searched user by name' do
+    #prepare
+    let!(:user) { create(:user, name: 'Dan') }
+    let!(:user1) { create(:user, name: 'Edimo') }
+    let!(:user2) { create(:user, name: 'Edmilson') }
+
+    it 'finds by name Ed' do
+      #action
+      get :index, params: { search: 'Ed' }
+      #assert
+      assigns(:users).should eq([user1, user2])
+    end
+
+    it 'finds by name with da' do
+      #action
+      get :index, params: { search: 'Da' }
+      #assert
+      assigns(:users).should eq([user])
+    end
```

Comment:
we need the other test scenarios:
1) when does not find any user
2) when search is empty

Reply:
I still can't post the test covered, only when I'm going to pass the request tests to the controller

Reply:
![Screenshot from 2020-09-02 11-02-02](https://user-images.githubusercontent.com/54644738/91993573-e7f15f00-ed0b-11ea-829e-2f06af24947e.png)

Reply:
should be at the pr description